### ISSUE TITLE
Icon component typings and docs are obsolete

### ISSUE THREAD

**Issue by Ledzz** [pmndrs/uikit#66]
The component "Icon" is called "[SVGIconFromText](https://docs.pmnd.rs/uikit/getting-started/components-and-properties#svgiconfromtext)" in documentation, I guess it's something from past.
Also, IconProperties don't contain "color" property, but the component seems to be working fine with it.
Is there something I don't see? If not, I will prepare PR with some fixes soon.

**Comment by bbohlender** [comment]
The "InheritableIconProperties" contains the color property.

Yes, SVGIconFromText is incorrect. A PR for that would be awesome :)

**Comment by Ledzz** [comment]
You are right about the property, my mistake, I meant when you call `iconRef.current.setStyle({ color })` there's no such property but it works.

**Comment by Ledzz** [comment]
And sorry again, I'm really don't have any attention today:) the error is
` Type '{ color: number; }' is missing the following properties from type '{ text: string; svgWidth: number; svgHeight: number; children?: ReactNode; }': text, svgWidth, svgHeight` so I guess `RefAttributes<ComponentInternals<IconProperties>>` is not quite valid, it should be Partial

**Comment by bbohlender** [comment]
you're right :)
I'd appreciate a PR for the docs and type fixes (can both be in one PR), but I can also do it myself during the next days :)

**Comment by Ledzz** [comment]
I'll try to do it today)
